src/helper.sh

```diff
@@ -79,16 +87,16 @@ write_to_log() {
 	dest_dir=$1
 	dest_file=$2
 	dest_path=$dest_dir$dest_file
-	
+
 	shift # eat first argument
 	shift # eat second argument
-	
+
 	# create empty file (overwrite previous content if it exists)
-	> $dest_path
-	
+	: > "$dest_path"
```

Comment:
I was curious what the : does. Would you care to explain?
(You can ignore this comment)

Reply:
not much difference between the two except `: >` is more portable, ShellCheck prefers it over `>`, and it looks like a smiley face :)